### ISSUE TITLE
use proper semver, implement semantic-release, and have CI auto-bump versions

### ISSUE THREAD

**Issue by mrnerdhair** [shapeshift/hdwallet#429]
As a developer, I want CI to bump the package version and publish after a PR is accepted so that I can actually use the code somewhere. We should implement commitlint and semantic-release a la `lib`.

**Comment by millsmcilroy** [comment]
Agreed. `lerna publish` should run in CI after a merge to master and not from a local workstation.
